app/chapters/[slug]/done/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+import { SignUpModal } from "components/chapters/SignUpModal";
+import { BoxButton } from "components/ui/BoxButton";
+import { useState } from "react";
+
+export default function Finished() {
+
+    const [openModal, setOpenModal] = useState(false)
+
+    function onSaveProgress(data) {
+        window.localStorage.setItem('user', JSON.stringify({
+          publicKey: data.keyPair.publicKey,
+          privateKey: data.keyPair.privateKey,
+          avatar: data.avatar,
+          progress: {
+            chapter: 'transacting',
+            lesson: 'transacting-2',
+          }
+        }))
+        setOpenModal(false)
+    }
+
+    return (
+        <div className="flex w-screen grow bg-fixed bg-cover bg-no-repeat" style={{backgroundImage: "url('/assets/images/chapter-finish.jpg')"}}>
```

Comment:
The layout breaks on mobile - the screenshot below is on 390x844 (iPhone 12 Pro size). The image should also center on mobile, so the character remains visible.

![localhost_3000_chapters_chapter-1_done(iPhone 12 Pro)](https://user-images.githubusercontent.com/695901/212020738-d807fcf9-1a08-4f08-a6c9-3bebf0ce034e.png)